python/cuml/dask/cluster/kmeans.py

```diff
@@ -26,62 +27,50 @@ class KMeans(object):
     Multi-Node Multi-GPU implementation of KMeans
     """
 
-    def __init__(self, n_clusters=8, init="k-means||", verbose=0):
+    def __init__(self, n_clusters=8, max_iter=300, tol=1e-4,
+                 verbose=0, random_state=1, precompute_distances='auto',
+                 init='scalable-k-means++', n_init=1, algorithm='auto',
+                 client=None):
         """
         Constructor for distributed KMeans model
         :param n_clusters: Number of clusters to fit
         :param init_method: Method for finding initial centroids
         :param verbose: Print useful info while executing
```

Comment:
Missing parameters in the docstring, I believe all/most of them are just copying them over from single gpu kmeans